.github/workflows/integration-tests.yml

```diff
@@ -51,17 +51,43 @@ jobs:
         run: |
           pip freeze
       - name: Upload Example Plots Artifact
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
-          name: example-plots
+          name: example-plots-${{ matrix.example_name }}
           path: ${{ steps.example-run.outputs.plots-path }}
 
       - name: Upload Example Results
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
-          name: example-results
+          name: example-results-${{ matrix.example_name }}
           path: ~/pyglotaran_examples_results
 
+  collect-plot-artifacts:
+    name: "Collect plot artifacts and reupload as bundel"
+    runs-on: ubuntu-latest
+    needs: [run-examples]
+    steps:
+      - name: Download All Artifacts
+        uses: actions/download-artifact@v4
+        with:
+          path: example-plots
+          pattern: example-plots-*
+          merge-multiple: true
+
+      - name: Upload Example Plots Artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: example-plots
+          path: example-plots
+          overwrite: true
+
+      - name: Delete Intermediate artifacts
+        #  TODO: Switch back to GeekyEggo/delete-artifact after
```

Comment:
Create an issue for that so you are reminded in case it takes a while?

Reply:
I created issue #1439